.tugboat/config.yml

```diff
@@ -0,0 +1,11 @@
+services:
+  node:
+    image: tugboatqa/node:14
+    default: true
+    expose: 3001
+    commands:
```

Comment:
```suggestion
    commands:
      init:
        - mkdir -p /etc/service/node
        - echo "#!/bin/sh" > /etc/service/node/run
        - echo "npm --prefix=$TUGBOAT_ROOT run heroku-serve -- build/localhost -p 3001" >> /etc/service/node/run
        - chmod +x /etc/service/node/run
```

Reply:
This will add the npm command that starts the http-server into a runit script, so that it will run in the background